### DISCUSSION TITLE
Swap to vue-tsc?

### DISCUSSION THREAD

**Discussion by colinblaise** [vuejs/vue-cli#7258] [General]
Hi, I'm sticking with vue-cli for the time being since there's a decent amount of refactoring I need to accomplish before moving to vite.

However, I'd love to start using vue-tsc.  Is it on the roadmap to incorporate vue-tsc in vue-cli?

**Comment by Viggo95** [comment]
In what case do we need `vue-tsc`？

**Reply by colinblaise** [reply]
Honestly, I swapped to vite now so this isn't an issue for me anymore.  But I was under the impression that volar/ vue 2.7/ vue-tsc kind of all went hand in hand

**Reply by Viggo95** [reply]
I had concerned about vue-tsc before, but I never knew if it need to be used  in webpack.
